email_functions.py

```diff
@@ -1,43 +1,33 @@
-'''
-reading_mail() - takes no input and returns a dictonary with mail argumnets as dict['subject']  , dict['msg_body']
-send_mail() - takes 2 arguments - subject , message and returns 1 if message is sent successfully and 0 is a error occurs
-'''
-import gmail
-import smtplib
-import json
-from email.mime.text import MIMEText
+# email_functions.py
+
 import Logger
 import os
+import requests
 
 logger = Logger.Logger(name='RunLog')
-f_email = os.getenv('SENDER_EMAIL_ID')
-t_email = os.getenv('SENDEE_EMAIL_ID')
-passwd = os.getenv('EMAIL_PASS')
+key = os.getenv('MAILGUN_API_KEY')
+sender = os.getenv('SENDER_EMAIL_ID')
+recipient = os.getenv('RECIPIENT_EMAIL_ID')
+sandbox = os.getenv('MAILGUN_SANDBOX_URL')
```

Comment:
This is actually the "MAILGUN_DOMAIN". We were using the sandbox domain, not anymore.